src/rules/at-rule-blacklist/README.md

```diff
@@ -0,0 +1,47 @@
+# at-rule-blacklist
+
+Specify a blacklist of disallowed at-rules.
+
+```css
+    @keyframes name {}
+/** ↑
+ * At-rules like this */
+```
+
+## Options
+
+`array`: `"["array", "of", "unprefixed", "at-rules"]`
+
+Given:
+
+```js
+["extend", "keyframes"]
+```
+
+The following patterns are considered warnings:
+
+```css
+a { @extend placeholder; }
+```
+
+```css
+@keyframes name {
+  from { top: 10px; }
+  to { top: 20px; }
+}
+```
+
+```css
+@-moz-keyframes name {
+  from { top: 10px; }
+  to { top: 20px; }
+}
+```
+
+The following patterns are *not* considered warnings:
+
+```css
+@mixin name ($p) {
+  ...
```

Comment:
I thought it would be great to have all these things in the docs for contributors. To be aware of them beforehand, not fixing one by one in the PRs ;)